ssl/d1_lib.c

```diff
@@ -475,13 +475,6 @@ int DTLSv1_listen(SSL *s, BIO_ADDR *client)
         return -1;
     }
 
-    /*
-     * We only peek at incoming ClientHello's until we're sure we are going to
-     * to respond with a HelloVerifyRequest. If its a ClientHello with a valid
-     * cookie then we leave it in the BIO for accept to handle.
-     */
-    BIO_ctrl(SSL_get_rbio(s), BIO_CTRL_DGRAM_SET_PEEK_MODE, 1, NULL);
```

Comment:
Since you remove all users of BIO_CTRL_DGRAM_SET_PEEK_MODE, should we get rid of it too?
Though, I don't whether BIO_CTRL_DGRAM_SET_PEEK_MODE is part of the public visiable ABI.

Reply:
It's part of the public ABI so cannot be removed